ansible/roles/enroot/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+enroot_ver: 3.4.1
+enroot_pkg_ver: 1
+enroot_deb: enroot_{{ enroot_ver }}-{{ enroot_pkg_ver }}_{{ deb_architecture.stdout }}.deb
+enroot_caps_deb: enroot+caps_{{ enroot_ver }}-{{ enroot_pkg_ver }}_{{ deb_architecture.stdout }}.deb
+enroot_rh: enroot-{{ enroot_ver }}-{{ enroot_pkg_ver }}.el8.{{ ansible_architecture }}.rpm
```

Comment:
I think `_rpm` would be clearer to most readers, especially now that Rocky and Alma are diverging from RedHat proper

Reply:
Done.